runtime/rasdump/dmpsup.c

```diff
@@ -73,6 +73,7 @@ static const J9RASdefaultOption defaultAgents[] = {
 	{ "heap",    "events=systhrow,range=1..4,filter=java/lang/OutOfMemoryError" },
 	{ "java",    "events=gpf,range=1..0" },
 	{ "java",    "events=user,range=1..0" },
+	{ "java",    "events=sigusr,range=1..0" },
```

Comment:
Again, we need to think about whether we should commandeer `SIGUSR2` - applications might already be using it for some other purpose.